src/routes/MenuRoutes.js

```diff
@@ -0,0 +1,18 @@
+import {Routes, Route} from "react-router-dom";
+import {Home} from "../pages/Home";
```

Comment:
- In this component, instead of using multiple import statements for importing pages, you can create a separate common-export file for pages from where you can export all the pages.

- It will reduce the line of code 

e.g: ```import {Home, Login, Signup} from "../pages/pages-common-export.js"```

Reply:
Great suggestions 👍